Makefile

```diff
@@ -27,5 +27,14 @@ eunit:
 qc:
 	$(REBAR) qc
 
+AFL_FUZZ = afl-fuzz
+AFL_CC = afl-gcc
+AFL_CXX = afl-g++
+FUZZ_SKELETON = fuzz_skeleton
 
-.PHONY: all compile get-deps clean check eunit qc check_valgrind
+fuzz:
+	CC=$(AFL_CC) CXX=$(AFL_CXX) $(REBAR) compile
```

Comment:
This should probably `clean` as well, since the NIF won't get rebuilt with afl information if it's already been compiled.